### PR TITLE
Update konnector configuration and manifest

### PR DESCRIPTION
**Pull request by gregorylegarec**
The goal is to pass the `konitor check` but I delayed the travis part to later, as I am not sure yet how we should publish this konnector on the registry.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "extends": ["eslint-config-cozy-app"]
```
Comment (CPatchane) [inline .eslintrc.json]: You can just put `"extends": ["cozy-app"]`

**Review by CPatchane — APPROVED** [review]
LGTM, just a detail 👍

File: .eslintrc.json (hunk 1 above)
Comment (gregorylegarec) [reply .eslintrc.json]: Thanks, I will also update this PR https://github.com/konnectors/konitor/pull/75

Do you have the commit or the pull-request where this string changed ?

Comment (CPatchane) [reply .eslintrc.json]: This is the way eslint works, you can ommit the `eslint-config-` before

Comment (gregorylegarec) [reply .eslintrc.json]: Thanks, I updated my PR https://github.com/konnectors/konitor/pull/75/commits/77c5c6a1b6176071f3d69f29a1ec714141c09d02

**Pull request merged**